Schemata/sarif-2.0.0-csd.2.beta.2018-10-10.json

```diff
@@ -296,7 +296,7 @@
         },
 
         "results": {
-          "description": "An array of locations of files containins arrays of run.result objects to be merged with the root log file.",
+          "description": "An array of locations of files containing arrays of run.result objects to be merged with the root log file.",
```

Comment:
As this is purely descriptive, I'd hope it's possible to fix, even if this field was retroactively dropped. That said, I can of course teach my tooling to ignore stale files.